loops/task_03.py

```diff
@@ -0,0 +1,6 @@
+sum = 0
+for num in range(10, 30, 3):
```

Comment:
30 should be inclusive)

```suggestion
for num in range(10, 31, 3):
```